homeassistant/components/modbus.py

```diff
@@ -33,10 +34,12 @@
 _LOGGER = logging.getLogger(__name__)
 
 SERVICE_WRITE_REGISTER = "write_register"
+SERVICE_WRITE_REGISTERS = "write_registers"
 
 ATTR_ADDRESS = "address"
 ATTR_UNIT = "unit"
 ATTR_VALUE = "value"
+ATTR_VALUES = "values"
```

Comment:
You could consider keeping the use of `value` and check for a list on the passed value, instead of adding another very similar attribute, which could lead to confusion and typos.